liquibase-snowflake/src/main/java/liquibase/sqlgenerator/core/SetColumnRemarksGeneratorSnowflake.java

```diff
@@ -12,6 +20,21 @@ public int getPriority() {
 
     @Override
     public boolean supports(SetColumnRemarksStatement statement, Database database) {
-        return database instanceof SnowflakeDatabase;
+        if (database instanceof SnowflakeDatabase) {
+            if (statement.getColumnParentType() != null) {
+                // Snowflake doesn't support setting the column remarks on a view.
+                return statement.getColumnParentType() != ColumnParentType.VIEW;
+            } else {
+                // Check if we're trying to set the column remarks on a view, and if so, note that this is not supported.
+                try {
+                    List<Map<String, ?>> viewList = Scope.getCurrentScope().getSingleton(ExecutorService.class).getExecutor("jdbc", database).queryForList(
+                            new RawSqlStatement("SHOW VIEWS LIKE '" + statement.getTableName() + "'"));
```

Comment:
Maybe this could be replaced by String.format() for better readability?

Reply:
Done